Source/ImageGlass/frmMain.cs

```diff
@@ -108,6 +109,12 @@ public partial class frmMain: Form {
         // slideshow countdown interval
         private uint _slideshowCountdown = 5;
 
+        // slideshow image alert counter
+
+        private uint _imgChangeAlertCounter = Configs.numImgChangeAlert;
+
+        private bool _chkimgChangeAlert = Configs.IsImgChangeAlert;
```

Comment:
Currently `chk` is referring to Checkbox component

```suggestion
        private bool _checkImageChangeAlert = Configs.IsImageChangeAlert;
```

Reply:
done